terraform/modules/application/role_task_execution.tf

```diff
@@ -0,0 +1,64 @@
+data "aws_iam_policy_document" "task_execution_assume_role" {
```

Comment:
No changes, I've just moved this into a separate file because it was getting too long for one.